.github/workflows/gh-ci.yaml

```diff
@@ -158,7 +158,7 @@ jobs:
           environment-file: docs/requirements.yaml
           add-pip-as-python-dependency: true
           architecture: x64
-          miniforge-variant: Mambaforge
+          miniforge-version: "latest"
           use-mamba: true
           channels: conda-forge, defaults
```

Comment:
Can we remove `defaults`? It's problematic with the whole anaconda license stuff.